craft_cli/messages.py

```diff
@@ -571,6 +571,22 @@ def verbose(self, text: str) -> None:
             use_timestamp = True
         self._printer.show(stream, text, use_timestamp=use_timestamp)
 
+    @_active_guard()
+    def warning(self, text: str, *, prefix: str = "WARNING: ") -> None:
```

Comment:
I think `WARNING` might be too harsh. How about `Warning`?

Also this parameter needs to be documented in the docstring

Reply:
(I'm happy with this change, but also pointing out that the text was described in the bounty :-) )